package.js

```diff
@@ -2,7 +2,7 @@
 
 Package.describe({
   name: 'communitypackages:react-router-ssr',
-  version: '3.0.4',
+  version: '3.0.5',
```

Comment:
This will have to be a major version release due to the breaking changes in react-router